src/main/webapp/resources/js/pages/analysis/components/sistr/CgMlst.jsx

```diff
@@ -24,11 +24,11 @@ export default function CgMlst({ sistrResults }) {
       },
       {
         title: i18n("AnalysisSistr.percentMatching"),
-        desc: `${getPercentage(sistrResults.cgmlst_distance).toString()}%`
+        desc: sistrResults.cgmlst_distance !== null ?`${getPercentage(sistrResults.cgmlst_distance).toString()}%` : ""
```

Comment:
Keeping on with random comments, we need to pay better attention to internationalization of numbers.  See [MDN Intl.NumberFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/NumberFormat)

Reply:
Good idea